package.json

```diff
@@ -4,8 +4,8 @@
   "description": "a documentation generator",
   "main": "index.js",
   "scripts": {
-    "test": "eslint index.js test/*.js && tape test/*.js",
-    "cover": "istanbul cover tape test/*.js --dir $CIRCLE_ARTIFACTS"
+    "test": "eslint index.js test/*.js test/filter/*.js && tape test/*.js test/filter/*.js",
```

Comment:
@tmcw Is there a way to set this up so that lint/test/cover all infer the sources (or at least use a shared definition)?